bindings/nodejs/lib/types/block/output/output.ts

```diff
@@ -126,8 +126,18 @@ abstract class CommonOutput extends Output /*implements ICommonOutput*/ {
  * Basic output.
  */
 class BasicOutput extends CommonOutput /*implements IBasicOutput*/ {
-    constructor(amount: bigint, unlockConditions: UnlockCondition[]) {
+    /**
+     * The amount of (stored) Mana held by the output.
+     */
+    readonly mana: u64;
+
+    constructor(
+        amount: bigint,
+        unlockConditions: UnlockCondition[],
+        mana: u64,
```

Comment:
Can you put mana right after amount in the constructors please ? More consistent with the TIP order